readme.org

```diff
@@ -78,6 +78,8 @@ the compilation database in the build directory.
 For all other cases, simple makefiles, scons, waf e.t.c. you have to provide the build command here making sure that it will
 provide visible compile output. You can simply add the =--dry-run= option in most build systems as this command should be used
 just to create the compilation database. The output will be parsed by malinka and the compilation database will be created.
+** compile-db-cmd
+This is the command tells malinka how to generate compilation database exactly. And =compile-db-cmd= takes precedence over configure-cmd. When =configure-cmd= is not enough to generate correct compilation database, you should use =compile-db-cmd=. One example is using ninja or bear to generate compilation database. You should provide a command that could always generate compilation database correctly. For example, you may add =bear make -B= to make sure it always generate compilation database whenever current project has been already built.
```

Comment:
`This is the command tells malinka how to generate compilation database exactly` -> `This command instructs malinka in what way to generate the compilation database`.

`And =compile-db-cmd= takes precedence over configure-cmd` -> `=compile-db-cmd= takes precedence over configure-cmd`

`is not enough to generate correct compilation database` -> ` is not enough to generate a correct compilation database`

`One example is using ninja` -> `One example is when using ninja`

`to generate compilation database` -> `to generate a compilation database`

`a command that could always generate compilation database` -> `a command that could always generate a compilation database`

`make sure it always generate compilation database` -> `make sure it always generate a compilation database`